tests/test_result.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
 
-from typing import Callable
+from typing import Callable, List
```

Comment:
With the `from __future__ import annotations` already at line 1, you should be able to use `list[...]` directly and no need to import `List`

Reply:
I wasn't sure if 3.8 has already added this :) 
Done